src/Translator.php

```diff
@@ -90,7 +90,8 @@ public function __construct(
      */
     protected function getMessage($key)
     {
-        if ($message = ($this->package->getMessage($key))) {
+        $message = $this->package->getMessage($key);
+        if ($message) {
```

Comment:
I'm actually wondering how this test should be. Should we differentiate if `$message = null` of `$message = false` and `$message = ''` ?